src/cli.js

```diff
@@ -92,7 +92,11 @@ cmd.option('apm.ens-registry', {
   description: 'Address of the ENS registry',
   default: require('@aragon/aragen').ens
 })
-cmd.group(['apm.ens-registry', 'eth-rpc'], 'APM:')
+cmd.option('legacy-role', {
```

Comment:
As commented above it is better to just remove the option and always check the role hash in the contract. However this option would have been better located in the command options directly, as it doesn't need to be a global CLI setting.

https://github.com/aragon/aragon-cli/blob/a7822caef299ea120768d5526b495635ba304b0a/src/commands/apm_cmds/grant.js#L10-L14